engine/distconsts/consts.go

```diff
@@ -19,7 +19,6 @@ const (
 
 const (
 	AlpineVersion = "3.20.1"
-	AlpineImage   = "alpine:" + AlpineVersion
```

Comment:
What's the reasoning for getting rid of this? Can we at minimum maintain usage of the `AlpineVersion` const elsewhere like in `core/integration/images.go` + `core/terminal.go`? It makes it much easier to update everywhere and we haven't had any needs for different alpine versions in different places.

Reply:
The idea is that the alpine module should be the place where alpine version is upgraded.

- `core/integration/images.go` should ideally use the alpine module also, but I don't dare touch it for now
- `core/terminal.go` is a special case, there's no reason for that feature to use the same alpine version as our CI pipelines - or even to couple its use of alpine to our CI pipeline's use of alpine. For example if we switch our CI build to wolfi, does that mean we should also move everyone's default terminal to wolfi? It seems weird.

Reply:
Generally relying on `const` and `distconsts` is at odds with making modules the source of truth for composing neatly encapsulated devops tasks.

Reply:
The reasoning is more that if there is a CVE or something in alpine it's easier to only have to remember there's one place to update it. Generally the container used for interactive terminals shouldn't be something users rely on toe be stable, at least in the current state of things where it's just a default one we choose (IMO).

It's not a huge deal either way, this is splitting hairs in obscure situations.

Reply:
Yeah in the case of terminal, in a perfect world I would advocate for depending on a module also (either the alpine module or another), to centralize updates as you say. But since it's in the core schema, that's not an option of course.

Maybe we change the core terminal image to target alpine latest?

I hear you re: splitting hairs :) Thanks for being open minded